app/config/rewriters/EndpointSummaryGroupRewriter.scala

```diff
@@ -22,93 +22,33 @@ import config.rewriters.DocumentationRewriters.CheckAndRewrite
 
 import javax.inject.{Inject, Singleton}
 
-// TODO repurpose this rewriter to use "maybeTestOnly" Handlebars helper etc
-
 /** For the OAS workaround where the "grouped endpoints" yaml file (e.g. employment_expenses.yaml) must include the matching summary text for each
   * endpoint. This rewriter checks and rewrites each endpoint summary in the group file.
```

Comment:
Probably need to update the description a little bit, as it no longer checks each endpoint summary, it's purely dependent on the handlebars stuff